src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -172,14 +186,25 @@ public void run() {
         }
       });
     } catch (final Exception e) {
-      // just don't show the wait window
+      ClientLogger.logQuietly(e);
+      if(s_waitWindow != null ) {
+        s_waitWindow.dispose();
+      }
     }
     setupProxies();
     showMainFrame();
-    // lastly, check and see if there are new versions of TripleA out
-    checkForUpdates();
+    checkForGameEngineUpdates();
   }
 
+  /**
+   * Used to set the object that will handle uncaught exceptions, generally these are exceptions
+   * thrown from the Swing and other threads. Generally useful to help test.
+   */
+  public static void setErrorHandler(UncaughtExceptionHandler handler) {
```

Comment:
Realistically, this should never change or be set.  Besides, if you have already passed the exceptionHandler to something, then setting it would not update that.  I think we don't need this method.